internal/impl/kafka/enterprise/redpanda_migrator_bundle_output.tmpl.yaml

```diff
@@ -134,29 +119,15 @@ mapping: |
         cases:
           - check: metadata("input_label") == "redpanda_migrator_input"
             output:
-              fallback:
-                - label: %s_redpanda_migrator_output
-                  redpanda_migrator: %s
-                  processors:
-                    - mapping: |
-                        meta input_label = deleted()
-                # TODO: Use a DLQ
-                - drop: {}
-                  processors:
-                    - log:
-                        message: |
-                          Dropping message: ${! content() } / ${! metadata() }
+              label: %s_redpanda_migrator_output
+              redpanda_migrator: %s
+              processors:
+                - mapping: |
+                    meta input_label = deleted()
           - check: metadata("input_label") == "redpanda_migrator_offsets_input"
             output:
-              fallback:
-                - label: %s_redpanda_migrator_offsets_output
-                  redpanda_migrator_offsets: %s
-                # TODO: Use a DLQ
-                - drop: {}
-                  processors:
-                    - log:
-                        message: |
-                          Dropping message: ${! content() } / ${! metadata() }
```

Comment:
Same here (it's the same logic, just repeated for the case when we don't use schemas).